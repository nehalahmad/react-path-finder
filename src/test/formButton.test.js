import React from 'react';
import MapButton from '../components/formButton';
import { shallow } from 'enzyme';

describe("<MapButton />", () => {
  it("submit button renders", () => {
      const SubmitButtonObj = shallow(<MapButton buttonType="submit" buttonLabel="Submit" isPrimary />);
      expect(SubmitButtonObj).toMatchSnapshot();
      expect(SubmitButtonObj.text()).toEqual("Submit");

      SubmitButtonObj.simulate("click");
  });

  it("reset button renders", () => {
    const ResetButtonObj = shallow(<MapButton buttonType="reset" buttonLabel="Reset" />);
    expect(ResetButtonObj).toMatchSnapshot();
    expect(ResetButtonObj.text()).toEqual("Reset");

    ResetButtonObj.simulate("click");
  });
});