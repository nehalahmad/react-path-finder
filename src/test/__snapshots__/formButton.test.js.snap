// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MapButton /> reset button renders 1`] = `ShallowWrapper {}`;

exports[`<MapButton /> submit button renders 1`] = `ShallowWrapper {}`;
